src/externals/bridge/bridge.service.ts

```diff
@@ -48,4 +103,23 @@ export class BridgeService {
       params,
     );
   }
+
+  async addStorage(
+    email: UserAttributes['bridgeUser'],
```

Comment:
We also want to provide an `addStorage` by UUID here, referrals is using it in old drive-web

Reply:
Good point

Reply:
Fixed